tests/lib/bin/session-tool

```diff
@@ -36,6 +55,68 @@ while [ $# -gt 0 ]; do
 	esac
 done
 
+case "$action" in
+	prepare)
+		# Make /var/lib/systemd writable so that we can get linger enabled.
+		# This only applies to Ubuntu Core 16 where individual directories were
+		# writable. In Core 18 and beyod all of /var/lib/systemd is writable.
+		case "$SPREAD_SYSTEM" in
+			ubuntu-core-16-*)
```

Comment:
you could also do `if is_core16_system` too

Reply:
Yes that's true. I prefer to avoid abstractions that are very simple. Checking on this is good enough. Originally this did contain quite a bit more cases but I managed to change the approach slightly and it was reduced to this.